### ISSUE TITLE
Account for dashes in project name

### ISSUE THREAD

**Issue by lacabra** [lacabra/submission-digitalpublicgoods#36]
When there is a dash `-` in the project name, such as in [Digital Services Portfolio - Agro Informatics](https://github.com/unicef/publicgoods-candidates/pull/605), [this line](https://github.com/lacabra/submission-digitalpublicgoods/blob/77812f3b59672985353c89482be5ea45a344eca1/pages/api/openPR.js#L27) incorrectly does the kebab transformation, resulting in (note the 3 consecutive `---`)
```
digital-services-portfolio---agro-informatics
```
whereas it should be:
```
digital-services-portfolio-agro-informatics
```
which [fails the CI](https://github.com/unicef/publicgoods-candidates/runs/2953397408?check_suite_focus=true#step:6:602)

For reference, the CI uses [this line](https://github.com/unicef/publicgoods-candidates/blob/de1b2ad7f60767d84890964e884df0fbbfb44cb9/scripts/check-filenames.bash#L14) to do the check, where you'd notice the relevant replacement pattern `'s/---/-/g'`

**Comment by nathanfletcher** [comment]
I am closing this because I addressed it in [#37](https://github.com/lacabra/submission-digitalpublicgoods/pull/37)

**Comment by nathanfletcher** [comment]
I am closing this because I addressed it in [#37](https://github.com/lacabra/submission-digitalpublicgoods/pull/37)

**Comment by nathanfletcher** [comment]
I am closing this because I addressed it in [#37](https://github.com/lacabra/submission-digitalpublicgoods/pull/37)
